tests/bin/set-wp-config.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
```

Comment:
I've not compared line by line but I know this is slightly different than the most recent version of this we've done. Can see that here: https://github.com/10up/insecure-content-warning/blob/develop/tests/bin/set-wp-config.js